taichi/codegen/llvm/kernel_compiler.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "taichi/codegen/kernel_compiler.h"
+#include "taichi/codegen/compiled_kernel_data.h"
+#include "taichi/runtime/llvm/llvm_context.h"
+
+namespace taichi::lang {
+namespace llvm {
```

Comment:
Should we change a name for this namespace because it may introduce conflicts with `::llvm`?

Reply:
Yes, good idea. Do you have some suggestions for the new name?

Reply:
maybe we can call it `LLVM` or `Llvm`?

Reply:
The #7580 has already been merged, which introduced the `namespace taichi::llvm`. Therefore, I plan to rename the "llvm" namespace to "LLVM" after merging these PRs (#7583~#7587).